### PR TITLE
Make http list available packages, not just most recent one

### PR DESCRIPTION
**Pull request by kvlung**
This change will allow users to install from any package version found on the server, not just the most recent one.

For steiza/simplepypi#10

### CODE REVIEW

File: simplepypi/bin/simplepypi [hunk 1]
```diff
@@ -105,10 +105,13 @@ class PypiPackageHandler(RequestHandler):
...
-            self.write(
-                '<html><body>{}</body></html>'.format(
-                    generate_package_link(package, versions[-1])
-                ))
+            self.write('<html><body>')
+            for version in versions:
+                self.write(
+                    '{}'.format(
+                        generate_package_link(package, version)
+                    ))
```
Comment (OskarGottlieb) [inline simplepypi/bin/simplepypi]: I guess there is no need for this formatting, given we're dealing only with output of the `generate_package_link` function?
```
self.write(generate_package_link(package, version))
```

**Comment by OskarGottlieb** [comment]
:+1: I had the same problem, would do the PR, but you were faster, only added a short comment.

File: simplepypi/bin/simplepypi (hunk 1 above)
Comment (kvlung) [reply simplepypi/bin/simplepypi]: Good call, that was dumb.

**Comment by steiza** [comment]
Looks good to me, thanks!

**Pull request merged**